modules/candidate_parameters/ajax/formHandler.php

```diff
@@ -363,32 +363,21 @@ function editConsentStatusFields($db, $user)
         exit;
     }
 
+    $currentUser = User::singleton();
+    $uid         = $currentUser->getData("UserID");
```

Comment:
getData shouldn't be used (in fact, it should probably be private.) $user->getUsername() is a better choice.